app/gui2/src/util/ast/abstract.ts

```diff
@@ -0,0 +1,165 @@
+/**
+ * # AST
+ *
+ * The goal of the new AST is to make expression edits simple.
```

Comment:
I'd drop a "new" from the docs here, since it will not be new anymore once we have it merged and used.